packages/flutter/lib/src/widgets/preferred_size.dart

```diff
@@ -111,7 +111,7 @@ abstract class PreferredSizeWidget implements Widget {
 ///    its preferred size.
 ///  * [AppBar] and [TabBar], which implement PreferredSizeWidget.
 class PreferredSize extends StatelessWidget implements PreferredSizeWidget {
-  /// Creates a widget that has a preferred size.
+  /// Creates a widget that has a preferred size that the parent can query.
```

Comment:
the parent => any ancestor

Reply:
How would someone other than a parent query it?

Reply:
i've added some more docs here but i'm pretty sure "parent" is actually right here.

Reply:
My mistake, parent is fine.